localstack-core/localstack/services/sqs/utils.py

```diff
@@ -128,6 +127,23 @@ def decode_receipt_handle(receipt_handle: str) -> str:
         )
 
 
+class ReceiptHandleInformation(NamedTuple):
+    identifier: str
+    queue_arn: str
+    message_id: str
+    last_received: str
+
+
+def extract_receipt_handle_info(receipt_handle: str) -> ReceiptHandleInformation:
+    try:
+        handle = base64.b64decode(receipt_handle).decode("utf-8")
+        return ReceiptHandleInformation(*handle.split(" "))
```

Comment:
maybe it's worth adding queue arn validation here and getting rid of decode_receipt_handle all together?
you could also simplify decode_receipt_handle by just saying `return extract_receipt_handle_info(...).queue_arn`

```python
parts = handle.split()
if len(parts) != 4:
   raise ValueError(...)
parse_arn(parts[1])
return ReceiptHandleInformation(*parts)